int64 size_x
int64 size_y
float64[] data